docs/operations-guide/running-metabase-on-docker.md

```diff
@@ -21,10 +21,35 @@ In its default configuration Metabase uses the local filesystem to run an H2 emb
 
 To persist your data outside of the container and make it available for use between container launches we can mount a local file path inside our container.
 
-    docker run -d -p 3000:3000 -v /tmp:/tmp -e "MB_DB_FILE=/tmp/metabase.db" --name metabase metabase/metabase
+    docker run -d -p 3000:3000 -v ~/metabase-data:/metabase-data -e "MB_DB_FILE=/metabase-data/metabase.db" --name metabase metabase/metabase
 
 Now when you launch your container we are telling Metabase to use the database file at `/tmp/metabase.db` instead of its default location and we are mounting that folder from our local filesystem into the container.
 
+### Getting your config back if you stopped your container
+
+If you ran and configured your Metabase using the local Database and then stopped the container, your data is still there unless you deleted the container with the `docker rm` command. All you need to do is
```

Comment:
'you ran' -> 'you have previously run'
'your data is' -> 'your data will still be'
'All you need to do is' -> 'To recover your previous configuration'